deployments/helm-chart/README.md

```diff
@@ -172,8 +172,7 @@ Parameter | Description | Default
 `controller.volumeMounts` | The volumeMounts of the Ingress controller pods. | []
```

Comment:
this doc https://github.com/nginxinc/kubernetes-ingress/blob/master/docs/content/installation/installation-with-helm.md also needs to be updated

Reply:
updated. I realised during the docs change I had excluded that folder to keep the project functional. It's fixed now but searching and other IDE features were ignoring that folder. Fixed now!